bus-mapping/src/rpc.rs

```diff
@@ -119,6 +119,20 @@ impl<P: JsonRpcClient> GethClient<P> {
             .map_err(|e| Error::JSONRpcError(e.into()))?;
         Ok(resp.0.into_iter().map(|step| step.result).collect())
     }
+
+    /// Calls `eth_getCode` via JSON-RPC returning a contract code
+    pub async fn get_code_by_address(
+        &self,
+        contract_address: Address,
+    ) -> Result<String, Error> {
```

Comment:
This endpoint returns the code as a hex string.  Could you decode the hex string and return a `Vec<u8>`?  This way we get the exact type we expect.

Reply:
I changed `String` to `Vec<u8>`.

Reply:
I think you forgot to decode the hex!  You can do that very easily using this type https://docs.rs/ethers-core/0.5.6/ethers_core/types/struct.Bytes.html like:
```rust
let code: Bytes = ...
// ...
code.to_vec()
```